tests/driver_at86rf2xx/cmd.c

```diff
@@ -248,7 +249,7 @@ static int send(int iface, le_uint16_t dst_pan, uint8_t *dst, size_t dst_len,
 {
     int res;
     netdev_ieee802154_t *dev;
-    const size_t count = 2;         /* mhr + payload */
+    size_t count = 2;         /* mhr + payload */
```

Comment:
How about static const instead?

Reply:
Well clang was complaining about the `const` so that's why I changed, since IMHO there's not necessary. What `static` would change?

Reply:
Static const means it won't take up stack space

Reply:
Well I get the original error if I change it:

```console
/Users/facosta/git/RIOT-OS/RIOT/tests/driver_at86rf2xx/cmd.c:253:25: error: variable length array folded to constant array as an extension
      [-Werror,-Wgnu-folding-constant]
    struct iovec vector[count];
                        ^~~~~
1 error generated.
```

Reply:
Can you make it a macro constant then instead?

```C
#define MAC_VECTOR_SIZE  (2)
```

Doesn't make much sense to use stack space for that constant anyway.